views/eApostilles/applicationSubmissionSuccessful.ejs

```diff
@@ -8,28 +8,24 @@
     <div class="container-confirmed govuk-!-margin-bottom-5">
         <h1 class="heading-xlarge confirmed">Application submitted</h1>
         <div class="heading-large govuk-!-font-weight-regular govuk-!-margin-top-0 govuk-!-margin-bottom-0">
-            Application reference number
+            Application reference
         </div>
         <div class="heading-large govuk-!-margin-top-0 govuk-!-margin-bottom-6">
-            <strong><%= applicationId %></strong>
+            <strong class="bold-text"><%= applicationId %></strong>
```

Comment:
https://design-system.service.gov.uk/styles/typography/font-weight/index.html govuk have a lot of helpers/utilities for this sort of stuff